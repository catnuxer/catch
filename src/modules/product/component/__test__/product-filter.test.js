import React from "react";
import serializer from "jest-emotion";
import renderer from "react-test-renderer";
import ProductFilter from "../product-filter";

expect.addSnapshotSerializer(serializer);

describe("product-filter test", () => {
  it("snappshoot", () => {
    expect(renderer.create(<ProductFilter />).toJSON()).toMatchSnapshot();
  });
});
