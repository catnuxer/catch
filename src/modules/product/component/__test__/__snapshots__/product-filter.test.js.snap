// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`product-filter test snappshoot 1`] = `
.emotion-0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  width: 100%;
  height: 100px;
  background: #f0f0f0;
}

<div
  className="emotion-0"
/>
`;
